README.md

```diff
@@ -2,22 +2,17 @@
 [![API Docs](https://img.shields.io/badge/API%20Docs-site-e140ad.svg)](https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/ccd-definition-designer-api.json)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Build Status](https://api.travis-ci.org/hmcts/ccd-definition-designer-api.svg?branch=master)](https://travis-ci.org/hmcts/ccd-definition-designer-api)
-[![Docker Build Status](https://img.shields.io/docker/build/hmcts/ccd-definition-designer-api.svg)](https://hub.docker.com/r/hmcts/ccd-definition-designer-api)
 [![codecov](https://codecov.io/gh/hmcts/ccd-definition-designer-api/branch/master/graph/badge.svg)](https://codecov.io/gh/hmcts/ccd-definition-designer-api)
```

Comment:
This repository is not configured to publish to docker hub at the moment.  The current pipeline publishes the docker image to hmcts.azurecr.io/hmcts/ccd-definition-designer-api instead